### PR TITLE
Add setters for units and content on game structure

### PR DESCRIPTION
**Pull request by bronvic**
#38 

Now on connect you'll receive field with one unit and one wall on it!
Serialization improves:
- Option fields now do not serialize
- All keys and values are lowercased
- Redundant type of walls dismissed

### CODE REVIEW

File: backend/src/game.rs [hunk 1]
```diff
@@ -16,10 +19,33 @@ impl Game {
             field: Grid::new(row_n, col_n),
         }
     }
+
+    pub fn set_unit(&mut self, x: u32, y: u32, unit: Unit) -> Result<(), &str> {
```
Comment (Hairash) [inline backend/src/game.rs:23]: Change order to (y, x)

File: backend/src/game.rs [hunk 2]
```diff
@@ -16,10 +19,33 @@ impl Game {
...
+        match self.field.get_hex(x, y) {
+            Some(hex) => {
+                hex.unit = Some(unit);
+                Ok(())
+            }
+            None => Err("no hex"),
+        }
+    }
+
+    pub fn set_content(&mut self, x: u32, y: u32, content: Content) -> Result<(), &str> {
```
Comment (Hairash) [inline backend/src/game.rs:33]: Change order to (y, x)

File: backend/src/game_objects/grid.rs [hunk 3]
```diff
@@ -21,13 +21,17 @@ impl Grid {
 
         Grid { hexes }
     }
+
+    pub fn get_hex(&mut self, x: u32, y: u32) -> Option<&mut Hex> {
```
Comment (Hairash) [inline backend/src/game_objects/grid.rs:25]: Change order to (y, x)

File: backend/src/game.rs (hunk 1 above)
Comment (alex-eg) [reply backend/src/game.rs:23]: Better also rename vars to `row`, `col` then.

**Pull request merged**